src/random_ray/vtk_plot.cpp

```diff
@@ -0,0 +1,154 @@
+#include "openmc/random_ray/vtk_plot.h"
+#include "openmc/mgxs_interface.h"
+#include "openmc/output.h"
+#include "openmc/plot.h"
+
+namespace openmc {
+
+// Outputs all basic material, FSR ID, multigroup flux, and
+// fission source data to .vtk file that can be directly
+// loaded and displayed by Paraview.
+void plot_3D_vtk()
+{
+  // Rename .h5 plot filename(s) to .vtk filenames
+  for (int p = 0; p < model::plots.size(); p++) {
+    PlottableInterface* plot = model::plots[p].get();
+    plot->path_plot() =
+      plot->path_plot().substr(0, plot->path_plot().find_last_of('.')) + ".vtk";
```

Comment:
It's strange to me that we'd restrict the vtk voxel plots to only random ray. Really, this should just be handled as a separate case in the `Plot` class. It could check based on the file ending.

Reply:
It seems to me that the only shared code is the rgb data, which is given to you by `PlottableInterface`.

Reply:
I'm not sure I follow -- as far as I can see, there is no RGB data in the random ray plotter at all, as it is just outputting a .vtk file. Basically just a formatted binary 3D matrix of data.